### DISCUSSION TITLE
Paginate selected page button color ?

### DISCUSSION THREAD

**Discussion by dgillier** [rappasoft/laravel-livewire-tables#1138] [Q&A] [answered]
Hello,

While Tailwind is great, main drawback is there is no class to override...

I'd like to have a more visible color for the current selected page from text-gray-500 to text-yellow-500 but I can't find a way to do that...

Any suggestion ?

Thanks,
Denis

**Comment by lrljoe** [comment, marked as answer]
I've merged a PR into Develop from someone else #1096 which improves the visibility.  I'm just in the process of doing final tests across TW2/3/BS4/BS5, and expect to issue a PR into main/master later today.

In the meantime you could add something like this into your CSS file, this will target the active immediate child span, or any nav element.  This includes the pagination, but doesn't appear to directly impact any other DataTable element (you may have used the nav navigation elsewhere though, so use caution!).

```php
nav[role="navigation"] [aria-current]:not([aria-current="false"]) > span {
    color: red;
}
```

I'd recommend adding a custom class to the Component Wrapper (e.g. RappasoftDataTable) and using that to reduce the scope

**Comment by dgillier** [comment]
Great :)))
Thanks a lot !
